Index.html

```diff
@@ -1,9 +1,50 @@
-# 3.8-with-3.4_CMatthews
-Development of the 9 Lives Website
-<html>
-	<head>
-		<h1>Test</h1>
-		</head>
-	<body>
-		</body>
-	</html>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+  
+	<title>9 Lives Alchemy & Strange Curios</title>
+	
+	<meta charset="utf-8">
+	
+	<meta name="Keywords" content="html5, layout, CSS Grid System"/>
+	<meta name="Author" content="Conal Matthews"/>
+	<meta name="Description" content="CSS Grid System Layout Tutorial"/>
+	
+	<!-- Import code needed for website to be responsive -->
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+
+	<!-- import the webpage's stylesheet goes here -->
+	<link rel="stylesheet" href="css/style.css">
+	<link href="https://fonts.googleapis.com/css?family=Spartan&display=swap" rel="stylesheet">
+  </head> 
+
+<body>
+
+<div class="grid-container">
+	
+	<div class="header"><p>Header</p><img class = "img" img src = "images/logo.jpg" alt = "9 Lives Logo"></div>
```

Comment:
This is supposed to show the logo that has been designed for the site, however the image isn't loading and will be looked into.